db/migrate/20230404171053_create_likes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :author, foreign_key: { to_table: 'users' }
```

Comment:
- [ ] it should only be for the posts that has a foreign key called `authro_id` but for likes and comments it should be `user_id` as in the diagram

Reply:
Updated 👍